### DISCUSSION TITLE
Adding a v2 HetAtomProtomer hash like HetAtomTautomerv2?

### DISCUSSION THREAD

**Discussion by mcs07** [rdkit/rdkit#7250] [Ideas] [answered]
Would it make sense to have a corresponding HetAtomProtomerv2 hash type that uses the HetAtomTautomerv2 algorithm?

The `TautomerHashv2` C++ function already has a bool `proto` argument just like the original tautomer hash, so I guess this would just need a new `HashFunction` enum member with a corresponding entry in the switch statement that calls this with proto=true.

**Comment by greglandrum** [comment]
Yep, definitely makes sense, and it's a very easy one to add.

**Reply by greglandrum** [reply, marked as answer]
Here's the PR: https://github.com/rdkit/rdkit/pull/7253
Thanks for the suggestion!
